doc/cephfs/index.rst

```diff
@@ -42,9 +42,21 @@ Block Devices) and RGW (Ceph Object Storage Gateway).
 Using CephFS
 ============
 
-Using the Ceph File System requires at least one :term:`Ceph Metadata Server` in
-your Ceph Storage Cluster.
+Using CephFS with a running Ceph Storage Cluster requires at least one active
+:doc:`Metadata Server (MDS) daemon </cephfs/add-remove-mds>`, :doc:`creating
+the file system </cephfs/createfs>`, selecting a mount mechanism for clients
+(:doc:`FUSE </cephfs/fuse>` or :doc:`kernel driver </cephfs/kernel>`), and
+configuring :doc:`authentication credentials </cephfs/client-auth>` for
+clients.
 
+For setting up CephFS quickly, checkout the :doc:`CephFS Quick Start document
+</start/quick-cephfs>`.
+
+.. note::
+
+         If :doc:`cephx </rados/configuration/auth-config-ref>` authentication
+         is enabled, a secret key is required for mounting CephFS with FUSE. It
```

Comment:
Mounting CephFS using FUSE doesn't require a key. Also, judging from the [ceph-fuse man page](https://docs.ceph.com/docs/jewel/man/8/ceph-fuse/), there is no way to pass a key.

Reply:
Key needs to be be copied before mounting([fuse doc](https://docs.ceph.com/docs/master/cephfs/fuse/))

Reply:
According to [quick start ](https://docs.ceph.com/ceph-prs/30251/start/quick-cephfs/) there is a `-k` option for passing key with FUSE.

Reply:
> Key needs to be be copied before mounting([fuse doc](https://docs.ceph.com/docs/master/cephfs/fuse/))

Yes

Reply:
> Key needs to be be copied before mounting([fuse doc](https://docs.ceph.com/docs/master/cephfs/fuse/))

From user's perspective passing it is optional. So, saying that "a secret key is required for mounting CephFS with FUSE" can give a wrong impression. It's an internal detail now since the secret key is picked automatically.

Reply:
> According to quick start there is a -k option for passing key with FUSE.

Adding it to man page - https://github.com/ceph/ceph/pull/30561

Reply:
> > Key needs to be be copied before mounting([fuse doc](https://docs.ceph.com/docs/master/cephfs/fuse/))
> 
> From user's perspective passing it is optional. So, saying that "a secret key is required for mounting CephFS with FUSE" can give a wrong impression. It's an internal detail now since the secret key is picked automatically.

Since it is a note for requirements, I did not add details in what way key can be passed.  Here the note just informs user that key is required if they have cephx enabled.

Reply:
Ok, I see the point. Perhaps rewrite it to say that secret key needs to be copied on the client host? I couldn't get it by reading this note.

Reply:
I have updated the doc.